src/browser/transport.ts

```diff
@@ -2,8 +2,36 @@ import Base from './transports/base';
 import ChromiumIPC from './transports/chromium_ipc';
 import UnixDomainSocket from './transports/unix_domain_socket';
 import WMCopyData from './transports/wm_copydata';
+import { EventEmitter } from 'events';
 
 /**
  * Conveniently exports available transports in one bundle
  */
 export { Base, ChromiumIPC, UnixDomainSocket, WMCopyData };
+
+export class NamedOneToManyTransport extends EventEmitter {
+    protected _transport: Base;
+
+    constructor(private name: string) {
+        super();
+    }
+
+    protected construct(): Base {
```

Comment:
fat arrow to protect `this`?

Reply:
I just copied this over from PortDiscovery. can change to fat arrow if we want.

Reply:
⚓️ :neckbeard: